content/departments/cloud/technical-docs/operations.md

```diff
@@ -370,6 +398,56 @@ git prune && git fetch # check for errors
        - SRC_ENABLE_GC_AUTO=true
    ```
 
+### Grant Access to the Frontend
```

Comment:
@danieldides @rafax I think we are duplicating the work in this and [other PR](https://github.com/sourcegraph/handbook/pull/4396#event-7044277557) ?